ee/codegen/src/generators/nodes/inline-prompt-node.ts

```diff
@@ -47,12 +48,32 @@ export class InlinePromptNode extends BaseSingleFileNode<
           block.blockType === "FUNCTION_DEFINITION"
       );
 
-    statements.push(
-      python.field({
-        name: "ml_model",
-        initializer: python.TypeInstantiation.str(nodeData.mlModelName),
-      })
+    const mlModelAttr = this.nodeData.attributes?.find(
+      (attr) => attr.name === "ml_model"
```

Comment:
[nit] let's start defining these constants towards the top of the file, we may source them form elsewhere in the future